### DISCUSSION TITLE
Alerts show no data available but kibana and wazuh repot lots of them

### DISCUSSION THREAD

**Discussion by ukr42** [Security-Onion-Solutions/securityonion#5920] [Unsupported Versions] [answered]
Hello,

after upgrading to 2.3.80, alerts aren't shown anymore, instead it says 'no data available'.

Grid shows all nodes are up (distributed environment with manager, fleet, 2 sensors and search node) without error and up to date.

Kibana shows >7M events in the last hour, /nsm/wazuh/logs/alerts/alerts.log reports new alerts being detected every few seconds.

The only errors I could find so far where related to wazuh communication but only for one agent so these can probably neglected.

Since wazuh reports alerts in the manager logs, these should definitely show up in the soc?

version: 2.3.80 (updated from 2.3.70)
distributed environment
so-status: all services running

**Comment by TtomtomT** [comment]
I have the same problem on a standalone environment after upgrading. Kibana shows events but Alerts and Hunt are fully empty.

**Reply by tiger-greenhell** [reply]
I have the same problem on distributed environment with manager, 2 search nodes, 1 forward sensor node and 1 heavy node.

Grid shows all nodes are up and counts EPS for every node.

Kibana shows events as before.

version: 2.3.80 (updated from 2.3.40)
distributed environment
so-status: all services running

**Comment by tiger-greenhell** [comment, marked as answer]
I was able to fix this problem. I assumed that the deployment of RBAC could cause this problem. 

problem:  /opt/so/conf/soc/soc_users_roles: Is a directory

"rename" file /opt/so/conf/soc/soc_users_roles/soc_users_roles.tmp to /opt/so/conf/soc/soc_users_roles

and so-soc-restart

good luck

**Reply by TtomtomT** [reply]
I can confirm that this is a fix, all my alerts and events are visible in SOC now. Note that the 'rename file' section also deletes the 'soc_users_roles' directory and moves the file one directory upwards, it's not just renaming a file.
Thank you!

**Reply by ukr42** [reply]
That did the trick, thank you!

**Reply by dougburks** [reply]
We've added a fix for this in the upcoming 2.3.90 release:
https://github.com/Security-Onion-Solutions/securityonion/issues/5952
